### PR TITLE
Update draft comments according to the new design

### PR DESCRIPTION
**Pull request by stee1house**

### CODE REVIEW

File: pages/dao/[dao]/drafts/[draft]/index.ts [hunk 1]
```diff
@@ -24,10 +24,15 @@ export const getServerSideProps: GetServerSideProps = async ({
   const daoContext = await getDaoContext(accountId, daoId as string);
 
   const dao = daoContext?.dao;
+  let draft;
 
-  const draft = await draftService.getDraft(draftId, dao, accountId);
+  try {
```
Comment (AlexeyPonomarenko) [inline pages/dao/[dao]/drafts/[draft]/index.ts]: Is there sense to place this try-catch block inside of the `getDraft` method?

File: astro_2.0/features/Comments/hooks.ts [hunk 2]
```diff
@@ -121,21 +132,58 @@ export function useDraftComments(): {
...
+    [accountId, draftsService, fetchComments, pkAndSignature]
+  );
+
+  const dislikeComment = useCallback(
+    async (id: string, isDislike: boolean) => {
+      if (!pkAndSignature) {
+        return;
+      }
+
+      const { publicKey, signature } = pkAndSignature;
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Comments/hooks.ts]: I consider we can shorten this block to
```
const { publicKey, signature } = pkAndSignature || {};
```

File: astro_2.0/features/Comments/components/Comment/Comment.tsx [hunk 3]
```diff
@@ -83,14 +94,25 @@ export const Comment: FC<CommentProps> = ({
...
+          <div className={styles.likes}>
+            {dislikeAccounts.length}
+            <IconButton
+              size="medium"
+              disabled={!accountId}
+              icon={isDislikedByMe ? 'likeFilled' : 'like'}
+              className={cn(styles.likeIcon, styles.dislikeIcon)}
+              onClick={() => onDislike(id, isDislikedByMe)}
+            />
+          </div>
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Comments/components/Comment/Comment.tsx]: What do you think about bringing this block to a separate component (nested inside this directory) and reusing it? Just a thought. Not mandatory.

File: astro_2.0/features/Comments/components/CommentActions/CommentActions.tsx [hunk 4]
```diff
@@ -42,38 +39,37 @@ export const CommentActions: FC<Props> = ({
...
+      >
+        <Button
+          variant="transparent"
+          className={styles.button}
+          size="small"
+          onClick={handleDelete}
+        >
+          <Icon name="buttonDelete" />{' '}
+        </Button>
+      </Tooltip>
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Comments/components/CommentActions/CommentActions.tsx]: What do you think about bringing this block to a separate component (nested inside this directory) and reusing it? Just a thought. Not mandatory.

File: astro_2.0/components/ProposalCardRenderer/components/ProposalCard/components/DeleteDraftButton/DeleteDraftButton.tsx [hunk 5]
```diff
@@ -21,22 +23,33 @@ export const DeleteDraftButton: FC<DeleteDraftButtonProps> = ({
...
+  const [showModal] = useModal(ConfirmModal);
 
   const handleDeleteDraft = useCallback(async () => {
-    const publicKey = await nearService?.getPublicKey();
-    const signature = await nearService?.getSignature();
+    if (!pkAndSignature) {
+      return;
+    }
+
+    const { publicKey, signature } = pkAndSignature;
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/ProposalCardRenderer/components/ProposalCard/components/DeleteDraftButton/DeleteDraftButton.tsx:34]: I consider we can shorten it to 
```
const { publicKey, signature } = pkAndSignature || {};
```

File: astro_2.0/components/EditableContent/EditableContent.module.scss [hunk 6]
```diff
@@ -30,11 +30,33 @@ $border-radius: 3px;
...
+    background-color: var(--color-neutral-20);
+    font-family: monospace, monospace;
+    font-size: 12px;
+    margin: 4px 0;
+    padding: 5px 10px;
+  }
+
+  img {
+    max-width: 100%;
+  }
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/EditableContent/EditableContent.module.scss]: It looks like this block is being repeated multiple times. Can we put it in mixin and reuse it, e.g. like in the first example [here](https://sass-lang.com/documentation/at-rules/mixin)?

**Review by VERZUOL1 — DISMISSED** [review]

File: pages/dao/[dao]/drafts/[draft]/index.ts (hunk 1 above)
Comment (stee1house) [reply pages/dao/[dao]/drafts/[draft]/index.ts]: fixed

File: astro_2.0/features/Comments/hooks.ts (hunk 2 above)
Comment (stee1house) [reply astro_2.0/features/Comments/hooks.ts]: Fixed

File: astro_2.0/features/Comments/components/Comment/Comment.tsx (hunk 3 above)
Comment (stee1house) [reply astro_2.0/features/Comments/components/Comment/Comment.tsx]: Fixed

File: astro_2.0/features/Comments/components/CommentActions/CommentActions.tsx (hunk 4 above)
Comment (stee1house) [reply astro_2.0/features/Comments/components/CommentActions/CommentActions.tsx]: Fixed

File: astro_2.0/components/ProposalCardRenderer/components/ProposalCard/components/DeleteDraftButton/DeleteDraftButton.tsx (hunk 5 above)
Comment (stee1house) [reply astro_2.0/components/ProposalCardRenderer/components/ProposalCard/components/DeleteDraftButton/DeleteDraftButton.tsx:34]: Fixed

File: astro_2.0/components/EditableContent/EditableContent.module.scss (hunk 6 above)
Comment (stee1house) [reply astro_2.0/components/EditableContent/EditableContent.module.scss]: fixed

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**